app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+  def index
+    User.all
+  end
+
+  def show
+    @set_user ||= User.includes(:posts).find(params[:id])
+    @recent_posts = @set_user.recent_posts
+  end
```

Comment:
- [ ] Great progress! I noticed a small typo in your code. In the `show` action of the `UsersController`, you're using `@set_user.recent_posts`, but based on your user model, it should be `@set_user.most_recent_posts`.
![image](https://github.com/BilalLiaquat7/blogapp/assets/55840999/d4fac799-090a-4bb5-b54e-df85c3ad02d7)


Ensure that you use the correct method name to retrieve the most recent posts for each user. Keep up the good work! 🚀👍